### PR TITLE
[small]feature/309

### PR DESCRIPTION
**Pull request by kaminuma**
Replaced "Landrace" in the pull-down data that appears when selecting a cat with 'Mixed Breed'

### CODE REVIEW

File: src/main/resources/db/migration/V1.5__update_breed.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE `breed` set name='Mixed Breed' where id=41;
```
Comment (josdem) [inline src/main/resources/db/migration/V1.5__update_breed.sql:1]: Looks good 👍

**Review by josdem — APPROVED** [review]

File: src/main/resources/db/migration/V1.5__update_breed.sql (hunk 1 above)
Comment (kaminuma) [reply src/main/resources/db/migration/V1.5__update_breed.sql:1]: I have a specification question.
in local environment
When I selected Cat, I didn't go to look at the contents of the DB, https://vetlog.org/breed/list?type=Cat
I am accessing the uri, is this correct?

Comment (josdem) [reply src/main/resources/db/migration/V1.5__update_breed.sql:1]: > Looks good 👍



> I have a specification question. in local environment When I selected Cat, I didn't go to look at the contents of the DB, https://vetlog.org/breed/list?type=Cat I am accessing the uri, is this correct?

Hi @kaminuma, in your local environment it should be: http://localhost:8080/breed/list?type=Cat
This is the output I am getting:

```json
[
  {
    "id": 40,
    "name": "Siamese",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 41,
    "name": "Mixed Breed",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 42,
    "name": "British Shorthair",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 43,
    "name": "Persian",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 44,
    "name": "Maine Coon",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 45,
    "name": "Ragdoll",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 46,
    "name": "American Shorthair",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 47,
    "name": "Abyssinian",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 48,
    "name": "Burmese",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 49,
    "name": "Bengal",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 50,
    "name": "Sphynx",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 51,
    "name": "Birman",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 52,
    "name": "Scottish Fold",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  },
  {
    "id": 53,
    "name": "American Bobtail",
    "type": "CAT",
    "dateCreated": "2017-06-03T21:23:01"
  }
]
```
Let me know if you have more quesrtions.

Comment (kaminuma) [reply src/main/resources/db/migration/V1.5__update_breed.sql:1]: Thanks👍
Looks good！

The parts I was concerned about seem to be fine, so I'm okay with it.

Comment (josdem) [reply src/main/resources/db/migration/V1.5__update_breed.sql:1]: > Thanks👍 Looks good！
> 
> The parts I was concerned about seem to be fine, so I'm okay with it.

Sure, anytime. If you want to see the change from the UI make sure:
1. You change  `breedsByTypeUrl` in the https://github.com/josdem/vetlog-spring-boot/wiki/YAML-File to point it to your local: http://localhost:8080/breed/list
2. Go to create a Pet url: http://localhost:8080/pet/create (You can use a pre-defined development user)
3. username:  `josdem` password: `12345678` You can find this definition here: https://github.com/josdem/vetlog-spring-boot/blob/main/src/main/java/com/josdem/vetlog/config/Bootstrap.java

Comment (kaminuma) [reply src/main/resources/db/migration/V1.5__update_breed.sql:1]: By referring to the above, I will deepen my understanding of the vetlog specifications. Thank you for your help

Comment (josdem) [reply src/main/resources/db/migration/V1.5__update_breed.sql:1]: > By referring to the above, I will deepen my understanding of the vetlog specifications. Thank you for your help

Happy to help!

**Pull request merged**